tests/foreman/cli/test_http_proxy.py

```diff
@@ -272,6 +272,13 @@ def test_positive_assign_http_proxy_to_products(module_org, module_target_sat):
         }
     )
     assert 'Product proxy updated' in res
+    module_target_sat.wait_for_tasks(
+        search_query=(
+            f'Actions::Katello::Repository::Update and organization_id = {module_org.id}'
+        ),
+        poll_rate=10,
+        max_tries=5,
```

Comment:
`wait_for_tasks` must be part of `host_helpers>capsule_mixins.py`, if yes then I would suggest optional changes to swap the order of arguments.
```suggestion
        max_tries=5,
        poll_rate=10,
```

Reply:
Ok, swapped.